.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,3 +13,4 @@ Resolves #
 - [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
 - [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
 - [ ] `make vendor` does not cause changes.
+- [ ] Comments have been added or updated based on the standards set in `CONTRIBUTING.md` *Coding Style*.
```

Comment:
Could you make that a link to the `CONTRIBUTING.md#coding-style` directly so the section.
Please test if a `CONTRIBUTING.md...` in an issue also correctly links to the file.

Reply:
> Could you make that a link to the CONTRIBUTING.md#coding-style directly so the section.
Neat. I didn't know links could do that.